src/test/v1.2/advanced/fdc3.open.ts

```diff
@@ -20,63 +21,74 @@ export default () =>
+    it(AFailsToOpenB2Test, async () => {
       try {
-        await control.openMockApp("ThisAppDoesNotExist", "ThisIdDoesNotExist");
+        let targetApp: TargetApp;
+        targetApp = { name: "ThisAppDoesNotExist"};
+        await control.openMockAppNew(targetApp);
         assert.fail("No error was not thrown", openDocs);
       } catch (ex) {
         control.confirmAppNotFoundErrorReceived(ex);
       }
     });
 
-    const AOpensBWithContext2Test = "(AOpensBWithContext2) Can open app B from app A with context and AppMetadata (name) as target, app B adds generic listener";
-    it(AOpensBWithContext2Test, async () => {
+    const AOpensBWithContext1 = `(AOpensBWithContext1) Can open app B from app A with context and appName as target, and app B listenss to that same context`;
+    it(AOpensBWithContext1, async () => {
+      let context: Context, targetApp: TargetApp;
+      context = { type: "fdc3.instrument", name: "context" };
+      targetApp = openApp.c.name;
       const receiver = control.contextReceiver("context-received");
-      await control.openMockApp(openApp.c.name, undefined, "fdc3.instrument");
+      await control.openMockAppNew(targetApp, context );
       await control.validateReceivedContext(await receiver, "fdc3.instrument");
-      await control.closeMockApp(AOpensBWithContext2Test);
+      await control.closeMockApp(AOpensBWithContext1);
     });
 
-    const AOpensBWithContext3Test = "(AOpensBWithContext3) Can open app B from app A with context and AppMetadata (name and appId) as target, app B adds generic listener";
-    it(AOpensBWithContext3Test, async () => {
+    const AOpensBWithContext2 = "(AOpensBWithContext2) Can open app B from app A with context and AppMetadata (name) as target, app B adds generic listener";
+    it(AOpensBWithContext2, async () => {
+      let context: Context, targetApp: TargetApp;
+      context = { type: "fdc3.instrument", name: "context" };
+      targetApp = { name: openApp.c.name};
       const receiver = control.contextReceiver("context-received");
-      await control.openMockApp(openApp.c.name, openApp.c.id, "fdc3.instrument");
+      await control.openMockAppNew(targetApp, context );
       await control.validateReceivedContext(await receiver, "fdc3.instrument");
-      await control.closeMockApp(AOpensBWithContext3Test);
+      await control.closeMockApp(AOpensBWithContext2);
     });
 
-    const AOpensBMalformedContext = `(AOpensBMalformedContext) App B listeners receive nothing when passing a malformed context`;
-    it(AOpensBMalformedContext, async () => {
-      const receiver = control.contextReceiver("context-received", true);
-      await control.openMockApp(openApp.f.name, undefined, undefined, true, true);
-      await receiver;
-      await control.closeMockApp(AOpensBMalformedContext);
-    });
+    // const AOpensBMalformedContext = `(AOpensBMalformedContext) App B listeners receive nothing when passing a malformed context`;
```

Comment:
remove commented out code

Reply:
or - explain why it's commented - which is that this will be in a future version of the conformance framework. 
you could actually just move it into a v2.1 folder?